Makefile

```
@@ -1,41 +1,139 @@
 VERSION=$(shell git describe --tags --abbrev=0)
 COMMIT=$(shell git rev-parse --short HEAD)
+DOCKER_COMPOSE_FILE=./.devcontainer/docker-compose.yaml
 
 ifeq ($(VERSION),)
 	VERSION="0.0.0"
 endif
 
+# Run gocyclo to check the cyclomatic complexities of the top 10 most complex functions
 gocyclo:
 	go run github.com/fzipp/gocyclo/cmd/gocyclo@latest -top 10 -ignore "_test|contract_" ./
 
+# Run gocognit to check cognitive complexities of the top 10 most complex functions
 gocognit:
 	go run github.com/uudashr/gocognit/cmd/gocognit@latest -top 10 -ignore "_test|contract_" ./
 
-# sort struct fields to reduce memory usage
+# Sort struct fields to reduce memory usage
 align:
 	go run github.com/dkorunic/betteralign/cmd/betteralign@latest -apply ./...
 
+# Run go generate to process any //go:generate directives
 generate:
 	go generate ./...
 
+# Run golangci-lint to lint the whole codebase, ensuring code quality
 lint:
 	go mod tidy
 	go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.58.1 run
 
+# Run all tests with coverage and enabled race detection
 test:
 	go test -cover -race -v ./...
 
+# Start all docker services defined in the docker-compose file (located in the ./devcontainer directory)
+.PHONY: service_up
+service_up:
+	docker-compose -f $(DOCKER_COMPOSE_FILE) up -d
```

Because docker compose uses the directory name as the project name by default,  the project name `devcontainer` currently used for compose is very unintuitive.
```
Node on  387-add-make-commands [$?] via 🐳 desktop-linux via 🐹 v1.22.1 via 🐍 v3.12.2 
❯ docker compose ls
NAME                STATUS              CONFIG FILES
devcontainer        running(6)          /Users/rustist/Developer/RSS3Network/Node/.devcontainer/docker-compose.yaml
```

```suggestion
	docker-compose --project-name rss3-node -f $(DOCKER_COMPOSE_FILE) up -d
```

---

Addressed.